x-pack/plugins/security_solution/public/management/components/endpoint_responder/components/header_info/header_agent_info.tsx

```diff
@@ -55,6 +70,12 @@ export const HeaderAgentInfo = memo<HeaderAgentInfoProps>(
             </EuiFlexItem>
           </EuiFlexGroup>
         </EuiFlexItem>
+
+        {agentType && (
```

Comment:
FYI - I don't see a test file for this component and I have marked this one ++ the other two siblings (`<AgentInfo>` and the sentinelone version of it) for refactor. I think they can be unified into one single component now that we have a single way to show statuses for them. I will add a test file then.

_(see team issue 9783)_